common/script-variable-generator/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>script-variable-generator</artifactId>
+	<packaging>jar</packaging>
+	<name>Script Variable Generator Support</name>
+	<description>Script Variable Generator Support</description>
+
+	<parent>
+		<groupId>org.springframework.cloud.stream.module</groupId>
```

Comment:
actually doesn't seem like we should have this parent, given that it brings in dependencies